hypervisor/arch/x86/guest/vm_reset.c

```diff
@@ -49,6 +49,9 @@ void triple_fault_shutdown_vm(struct acrn_vcpu *vcpu)
 
 		/* Either SOS or pre-launched VMs */
 		get_vm_lock(vm);
+		if (is_rt_vm(vm)) {
```

Comment:
I am thinking we can avoid this condition check, and let the poweroff_rt_vm(vm) be called every case.

Reply:
Of course, in this case, the name of poweroff_rt_vm may be changed.  Even if we don't change, it is ok too.